pkgs/applications/misc/xygrib/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, bzip2, qtbase, qttools, libnova, proj, libpng, openjpeg } :
+
+stdenv.mkDerivation rec {
+  version = "1.2.6.1";
+  name = "xygrib-${version}";
+
+  src = fetchFromGitHub {
+    owner = "opengribs";
+    repo = "XyGrib";
+    rev = "v${version}";
+    sha256 = "0xzsm8pr0zjk3f8j880fg5n82jyxn8xf1330qmmq1fqv7rsrg9ia";
+   };
+
+  nativeBuildInputs = [ cmake qttools ];
+  buildInputs = [ bzip2 qtbase libnova proj openjpeg libpng ];
+  cmakeFlags = [ "-DOPENJPEG_INCLUDE_DIR=${openjpeg.dev}/include/openjpeg-2.3" ];
+
+  postInstall = ''
+    mkdir $out/bin
+    ln -s $out/XyGrib/XyGrib $out/bin/XyGrib
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://opengribs.org;
```

Hint: We're probably going to deprecate unquoted URL syntax with RFC 45.
```suggestion
    homepage = "https://opengribs.org";
```